### ISSUE TITLE
Unsafe use of $ in jquery.payment.js

### ISSUE THREAD

**Issue by swiderski** [stripe/jquery.payment#161]
Original issue in woocommerce. I don't know if you can do something about it:
https://github.com/woothemes/woocommerce/issues/8191

I don't know coffee script, so please excuse me.
In resulting js file in line 7 you have such code
`$ = jQuery;`
It is unsafe use of it, as later call of `jQuery.noConflict();`, that can occur in theme or plugin script, breaks this.
You should put your code in
```javascript
( function( $ ){
//your code
}( jQuery ) );
```

Is that possible?

**Comment by websiteduck** [comment]
I had the same problem and was able to wrap /lib/jquery.payment.js like this:
```
( function( $ ){

...

}(jQuery)).call(this);
```

**Comment by frankcortes** [comment]
In previous versions of this library, the `$` variable [was local](https://github.com/stripe/jquery.payment/blob/v1.0.3/lib/jquery.payment.js#L3) and it used the value as a reference of the current [jQuery](https://github.com/stripe/jquery.payment/blob/v1.0.3/lib/jquery.payment.js#L3). What happened? It resolved the problem. Nevertheless, @websiteduck's proposal will be cleaner.

**Comment by jamesreggio** [comment]
Apologies; fixed in 2c2f99b897b1d724b0b5d2353fe3d45eac917121.
